client/src/components/page/Sidebar/index.tsx

```diff
@@ -84,78 +85,78 @@ export const SidebarComponent: React.FC<SidebarProps> = (props: SidebarProps) =>
         className="menu-dark"
       >
         {ENABLE_PLANS && (
-          <Menu.SubMenu key="missions" icon={<DashboardOutlined />} title={MISSIONS}>
-            <Menu.Item key="active">
+          <Menu.SubMenu key={MISSIONS} icon={<DashboardOutlined />} title={MISSIONS}>
+            <Menu.Item key={`${MISSIONS}-${ACTIVE}`}>
               <Link to={ACTIVE_PLANS_LIST_VIEW_URL} className="admin-link">
                 {ACTIVE}
               </Link>
             </Menu.Item>
-            <Menu.Item key="draft">
+            <Menu.Item key={`${MISSIONS}-${DRAFT}`}>
               <Link to={DRAFT_PLANS_LIST_VIEW_URL} className="admin-link">
                 {DRAFT}
               </Link>
             </Menu.Item>
-            <Menu.Item key="complete">
+            <Menu.Item key={`${MISSIONS}-${COMPLETE}`}>
               <Link to={COMPLETE_PLANS_LIST_VIEW_URL} className="admin-link">
                 {COMPLETE}
               </Link>
             </Menu.Item>
-            <Menu.Item key="trash">
+            <Menu.Item key={`${MISSIONS}-${TRASH}`}>
               <Link to={TRASH_PLANS_LIST_VIEW_URL} className="admin-link">
                 {TRASH}
               </Link>
             </Menu.Item>
           </Menu.SubMenu>
         )}
-        <Menu.SubMenu key="admin" icon={<DashboardOutlined />} title={ADMIN}>
+        <Menu.SubMenu key={ADMIN} icon={<DashboardOutlined />} title={ADMIN}>
           {roles && roles.includes('ROLE_EDIT_KEYCLOAK_USERS') && (
-            <Menu.SubMenu key="users" title={USERS}>
-              <Menu.Item key={'list'}>
+            <Menu.SubMenu key={`${ADMIN}-${USERS}`} title={USERS}>
+              <Menu.Item key={`${ADMIN}-${USERS}-${USER_MANAGEMENT}`}>
```

Comment:
If we are changing `key`, then we also need to modify routes,
because both are dependent on each other for menu item to expand and work properly.

Reply:
@haseemVD The routes seem to be working fine so far, or what do you mean by "because both are dependent on each other for menu item to expand and work properly."?

Reply:
@kelvin-muchiri lets say we have to open user module having route `/admin/user/list`
Then the keys for menu item should look like this,

```
<Menu.SubMenu key={`admin`} icon={<DashboardOutlined />} title={ADMIN}>
   <Menu.SubMenu key={`user`} title={USERS}>
       <Menu.Item key={`list`}>
              <Link to={URL_USER} className="admin-link">
                  {USER_MANAGEMENT}
              </Link>
      </Menu.Item>
   </Menu.SubMenu>
</Menu.SubMenu>
```

Reply:
This is because `defaultOpenKeys` requires an array of keys, and we are extracting that from our URL and then creating an array. so we need to keep keys and url same, splitting with `/`

Reply:
@haseemVD Ok noted